src/openforms/js/components/formio_builder/builder.js

```diff
@@ -92,13 +92,23 @@ const FormIOBuilder = ({ configuration, onChange, onComponentMutated, forceUpdat
         }
     );
 
+    if (onComponentMutated) {
```

Comment:
But, why wouldn't it be passed to the constructor? This seems like a workaround on a deeper issue?

Reply:
Because this component is used both in the Form view and in the FormDefinition view. What `onComponentMutate seem` to do in the case of the Form view (see https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/admin/form_design/form-creation-form.js#L736 and https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/admin/form_design/form-creation-form.js#L290) is to check whether changes are needed for the Registration component. 

But this check doesn't need to happen for FormDefinitions that are being edited independently of the form.